<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRoboCmdClass</class>
 <widget class="QDialog" name="QRoboCmdClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Robot Commander</string>
  </property>
  <widget class="QPushButton" name="m_act1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Action 1</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_act2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_act3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Action 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_act4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_quit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="m_msg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>521</width>
     <height>71</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrobocmd.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_quit</sender>
   <signal>released()</signal>
   <receiver>QRoboCmdClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
